### ISSUE TITLE
Route with query params don't found

### ISSUE THREAD

**Issue by byluisro** [talentlessguy/tinyhttp#214]
**Describe the bug**

When make a get request with query params don't handle the app route instead log `404 NOT FOUND`.

Code:
```javascript
app.get('/element', function(req, res){
  res.send(req.query);
});
```

Logger output:
```javascript 
GET 404 Not Found /element?page=0&limit=5
```

Dependencies:
```json
"dependencies": {
    "@tinyhttp/app": "^1.1.6",
    "@tinyhttp/cors": "^1.1.0",
    "@tinyhttp/logger": "^1.1.6",
    "milliparsec": "^1.0.4"
  },
```

**Comment by talentlessguy** [comment]
Oopsie I'm sorry for this silly mistake, looks like i'm trying to match `req.url` and `mw.path`

Fixing now...

**Comment by talentlessguy** [comment]
Fixed in 1.1.7:

@byluisro please install new version:

```sh
pnpm i @tinyhttp/app@1.1.7
```

**Comment by byluisro** [comment]
> Fixed in 1.1.7:
> 
> @byluisro please install new version:
> 
> ```shell
> pnpm i @tinyhttp/app@1.1.7
> ```

Amazing, now it's working thanks
